### PR TITLE
feat: add PostgreSQL17beta3 test postgis image

### PR DESCRIPTION
**Pull request by thomasboussekey**

### CODE REVIEW

**Comment by thomasboussekey** [comment]
Hello @NiccoloFei ,

Thanks for working on this PR

Can you explain me which change you have performed on your commit 32d8103?
It looks very similar to the commit c0e2221.

**Comment by NiccoloFei** [comment]
@thomasboussekey Not working on this one yet, sorry. I've just hit the PR rebase button, no changes have been added. btw, have you tried building that Dockerfile? Because I doubt those additional extensions are already available for PG17.

**Comment by thomasboussekey** [comment]
@NiccoloFei, Thanks for your response!

I forgot to test during my first attempt.

I have just tested, and removed 2 unavailable packages:

* "postgresql-${PG_MAJOR}-pg-failover-slots"
* "postgresql-${PG_MAJOR}-pgrouting" 

Lost with the commits using `DCO` my last commit is :red_circle: :cry:

File: PostGIS/17/.versions.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "BARMAN_VERSION": "3.10.1",
+  "IMAGE_RELEASE_VERSION": "1",
+  "POSTGIS_IMAGE_LAST_UPDATED": "2024-07-22T19:00:00",
+  "POSTGIS_IMAGE_VERSION": "17beta2-master"
```
Comment (thomasboussekey) [inline PostGIS/17/.versions.json:5]:
```suggestion
  "POSTGIS_IMAGE_VERSION": "17beta3-master"
```

File: PostGIS/17/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+#
+FROM postgis/postgis:17beta2-master
+
+# Do not split the description, otherwise we will see a blank space in the labels
+LABEL name="PostgreSQL + PostGIS Container Images" \
+      vendor="The CloudNativePG Contributors" \
+      version="${PG_VERSION}" \
+      release="53" \
+      summary="PostgreSQL + PostGIS Container images." \
+      description="This Docker image contains PostgreSQL, PostGIS and Barman Cloud based on Postgres 17beta2-master."
```
Comment (thomasboussekey) [inline PostGIS/17/Dockerfile:25]:
```suggestion
      description="This Docker image contains PostgreSQL, PostGIS and Barman Cloud based on Postgres 17beta3-master."
```

File: PostGIS/Dockerfile-beta.template [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+# Do not split the description, otherwise we will see a blank space in the labels
+LABEL name="PostgreSQL + PostGIS Container Images" \
+      vendor="The CloudNativePG Contributors" \
+      version="${PG_VERSION}" \
+      release="53" \
+      summary="PostgreSQL + PostGIS Container images." \
+      description="This Docker image contains PostgreSQL, PostGIS and Barman Cloud based on PostGIS %%POSTGIS_IMAGE_VERSION%%."
+
+LABEL org.opencontainers.image.description="This Docker image contains PostgreSQL and Barman Cloud based on PostGIS %%POSTGRES_IMAGE_VERSION%%."
```
Comment (thomasboussekey) [inline PostGIS/Dockerfile-beta.template:27]:
```suggestion
LABEL org.opencontainers.image.description="This Docker image contains PostgreSQL and Barman Cloud based on PostGIS %%POSTGIS_IMAGE_VERSION%%."
```
@sxd, POSTGRES_IMAGE_VERSION is not set in this repository

**Review by sxd — APPROVED** [review]

**Review by NiccoloFei — APPROVED** [review]

**Pull request merged**